onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -1494,4 +1494,59 @@ TEST(CApiTest, ConfigureCudaArenaAndDemonstrateMemoryArenaShrinkage) {
   // (Memory arena for the CPU should not have been disabled)
   //  run_option.AddConfigEntry(kOrtRunOptionsConfigEnableMemoryArenaShrinkage, "cpu:0;gpu:0");
 }
-#endif
\ No newline at end of file
+#endif
+
+//#ifdef USE_TENSORRT
+// This test uses CreateTensorRTProviderOptions/UpdateTensorRTProviderOptions APIs to configure and create
+// a TensorRT Execution Provider
+TEST(CApiTest, TestConfigureTensorRTProviderOptions) {
+  const auto& api = Ort::GetApi();
+  OrtTensorRTProviderOptions* trt_options;
+  ASSERT_TRUE(api.CreateTensorRTProviderOptions(&trt_options) == nullptr);
+  std::unique_ptr<OrtTensorRTProviderOptions, decltype(api.ReleaseTensorRTProviderOptions)> rel_trt_options(trt_options, api.ReleaseTensorRTProviderOptions);
+
+#ifdef _WIN32
+  const char* engine_cache_path = ".\\trt_engine_folder";
```

Comment:
> const char* engine_cache_path = ".\\trt_engine_folder"; [](http://example.com/codeflow?start=1&length=56)

You do not need this part. Windows API accepts forward slashes.

Reply:
This part is removed.